nerdy/bin/Activate.ps1

```diff
@@ -0,0 +1,241 @@
+<#
+.Synopsis

```

Comment:
not quite sure what is it, but I think it's better to place this into `.gitignore`